doc/STRUCTURE.md

```diff
@@ -6,6 +6,7 @@
 
 ![Image](structure.png)
 
+Source: [draw.io](https://drive.google.com/file/d/1ZoVobQRdWoE8qrE-Ma1hhybDXxwNMAJN/view?usp=sharing)
```

Comment:
might be a better idea to store it directly in git.

Reply:
Indeed, 
I did update the PR ^^;